Makefile

```diff
@@ -15,13 +15,21 @@ GO_BIN ?= go
 TEST_COMMAND ?= $(GO_BIN) test
 
 # The compute tests can sometimes exceed the default 10m limit.
-TEST_ARGS ?= -timeout 15m ./{cmd,pkg}/...
+TEST_ARGS ?= -timeout 15m ./...
```

Comment:
ℹ️ Following my comment about switching to `cmd.exe` the shell expansion `{x, y}` doesn't work in `cmd.exe` and so we have to remove it. I'm not sure why we did this, maybe for performance reasons (e.g. narrow down the number of files the `go` toolchain has to lookup).